backend/database/seeders/SubmissionSeeder.php

```diff
@@ -31,10 +31,22 @@ public function run()
      *
      * @param int $id
      * @param string $title
-     * @return void
+     * @param array $data
+     * @return \Database\Seeders\App\Models\Submission
      */
-    protected function createSubmission($id, $title)
+    protected function createSubmission(int $id, string $title, array $data = [])
```

Comment:
I changed this up a little bit to accept data as an optional third parameter rather than requiring an extra save.